packages/components/src/internal/components/user/UserDetailsPanel.tsx

```diff
@@ -118,6 +127,21 @@ export class UserDetailsPanel extends React.PureComponent<Props, State> {
         );
     }
 
+    renderGroups(): JSX.Element {
```

Comment:
`JSX.Element`? should this be `ReactNode`?

Reply:
Neither 😄 , this method should be a component.

Reply:
Modularized the renderGroups fn into its own component—there's a bunch of renderX() fns on the page that I could also modularize, but since it's older code I'm leaving it for now. Happy to go back over them if we feel it'd be valuable though for cleanliness and consistency!